index.d.ts

```diff
@@ -516,7 +517,7 @@ export function isAxiosError<T = any, D = any>(payload: any): payload is AxiosEr
 
 export function spread<T, R>(callback: (...args: T[]) => R): (array: T[]) => R;
 
-export function isCancel(value: any): value is Cancel;
+export function isCancel(value: any): value is CanceledError<any>;
```

Comment:
Thanks for submitting this fix! Changing isCancel to check for `CanceledError<any>` makes sense. However, I think we can improve the typings further by making `isCancel` generic:

```suggestion
export function isCancel<T = any>(value: any): value is CanceledError<T>;
```

This allows `isCancel` to preserve the generic type parameter `T` from `CanceledError,` rather than hardcoding it to `any`.